style.css

```diff
@@ -0,0 +1,96 @@
+h1 {
+    text-align: center;
+    margin: 40px;
+}
+body {
+    font-family: "Arial", sans-serif;
+    font-size: 14px;
+    
+}
+.e1fqkh3o5{
+    background-color: white !important;
+}
+
+form {
+    width: 60%;
+    margin: 20px auto;
+    overflow: visible;
+    padding: 25px;
+}
+p {
+    font-weight: bold;
+}
+span .number-label {
+    padding: 8px 12px;
+    background-color: #dddddd;
```

Comment:
It looks like this is creating a grey everywhere between the tabs and below, which is looking messy. Can it be closer to the design, which seems to just be white?

Reply:
<img width="1257" alt="Screen Shot 2022-10-25 at 10 21 06 AM" src="https://user-images.githubusercontent.com/90473723/197840762-d8b3e090-f829-4449-9004-339cc6c3500d.png">

Reply:
May also be the one on line 57.

Reply:
I believe the grey color is #dddddd

Reply:
Similarly, I'm not positive if it would be here, but if we're going to use a background color please also adding padding on the right and left of the text (especially the tab text) so the text is not hitting the border. However, if the grey is removed, and it's white instead, you won't see a border, so it won't be necessary.

Reply:
i think it should be fixed now